src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinSpan.cs

```diff
@@ -281,33 +282,41 @@ public void Write(Utf8JsonWriter writer)
                 writer.WritePropertyName("tags");
                 writer.WriteStartObject();
 
+                var originalUICulture = Thread.CurrentThread.CurrentUICulture;
```

Comment:
I think it is work a comment that this is needed because of `string.Join` below.

Reply:
not just because of string.join. If u see, i removed the ToString from int and double, which will fall to the default case. but yes,  I will add some comments! Thank you!